### DISCUSSION TITLE
Swap Rofi for Walker?

### DISCUSSION THREAD

**Discussion by Jean-Paul76** [mylinuxforwork/dotfiles#514] [Q&A]
Anyone that's been able to replace Rofi with Walker?

https://github.com/abenz1267/walker

**Comment by mylinuxforwork** [comment]
@Jean-Paul76 Walker looks interesting. I can move the execution command for rofi into settings. Then rofi can be replaced with any launcher.

**Comment by Jean-Paul76** [comment]
That would be awesome, thank you!

Kind regards

Jean-Paul
________________________________
